docs/examples/paddle/paddle-various-readers.ipynb

```diff
@@ -0,0 +1,308 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Using Paddle DALI plugin: using various readers\n",
+    "\n",
+    "### Overview\n",
+    "\n",
+    "This example shows how different readers could be used to interact with Paddle. It shows how flexible DALI is.\n",
+    "\n",
+    "The following readers are used in this example:\n",
+    "\n",
+    "- MXNetReader\n",
+    "- CaffeReader\n",
+    "- FileReader\n",
+    "- TFRecordReader\n",
+    "\n",
+    "For details on how to use them please see other [examples](..)."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Let us start from defining some global constants"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# MXNet RecordIO\n",
+    "db_folder = \"/data/imagenet/train-480-val-256-recordio/\"\n",
```

Comment:
It would be better to use DALI_extra, like in https://docs.nvidia.com/deeplearning/sdk/dali-developer-guide/docs/examples/pytorch/pytorch-various-readers.html.

Reply:
done.